moped-editor/src/views/staff/helpers.js

```diff
@@ -57,7 +72,49 @@ const errorsToTranslate = {
     "password must only contain: a-z, A-Z, 0-9, and any of these special characters: _-!@%^~?.:&()[]$",
 };
 
-export const formatApiErrors = errorsArray =>
+export const formatApiErrors = (errorsArray) =>
   errorsArray
-    ? errorsArray.map(error => errorsToTranslate[error] || error).join(", ")
+    ? errorsArray.map((error) => errorsToTranslate[error] || error).join(", ")
     : null;
+
+/**
+ * Makes sure the password looks ok
+ * @returns {boolean}
+ */
+export const passwordLooksGood = (password) =>
+  new RegExp(
+    "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$"
+  ).test(password);
```

Comment:
ok, very out of scope, but, if we wanted to increase the min password length to, say, 12 chars, is that a matter of changing it here and the backend API? if it's minimal i think we should?

Reply:
This would be a small change that would easily roll into this PR. I'll do this before merging. 👍

Reply:
fantastic — thanks Mike!